commands/stop.go

```diff
@@ -66,6 +67,8 @@ func (cmd *Stop) StopOutrigger() error {
 		util.Command("sudo", "route", "-n", "delete", "-net", "172.17.42.1").Run()
 	}
 	color.Unset()
+	cmd.out.Success("Networking cleanup completed")
+	cmd.out.NoSpin()
```

Comment:
No need to `NoSpin()` here, it's handled in `BaseCommand.Success()` no?

Reply:
True.